Dispatchr/app/containers/App.js

```diff
@@ -20,16 +19,16 @@ const getSceneStyle = (props, computedProps) => {
   return style;
 };
 
-export default class App extends Component {
+class App extends Component {
   render() {
     return (
-      <Provider store={configureStore()}>
```

Comment:
If anything it was a style preference, it changes nothing. Just thought it would be neater to have just the routers in class App.